app/src/main/res/values/strings.xml

```diff
@@ -203,12 +203,11 @@
 
     <!-- About -->
     <string name="about_title">About</string>
-    <string name="about_summary">A backup application using Android\'s internal backup API.</string>
+    <string name="about_summary">A backup application using Android\'s internal backup API. It is Free Software, released under Apache 2 license.\n\nLike all software, Seedvault may contain bugs or vulnerabilities.</string>
     <string name="about_version">Version: %s</string>
-    <string name="about_license">License: <a href="https://www.apache.org/licenses/LICENSE-2.0">Apache2</a></string>
-    <string name="about_author">Written by: <a href="https://github.com/stevesoltys">Steve Soltys</a> and <a href="https://blog.grobox.de">Torsten Grote</a></string>
-    <string name="about_design">Design by: <a href="https://www.glennsorrentino.com/">Glenn Sorrentino</a></string>
-    <string name="about_sponsor">Sponsored by: <a href="https://www.calyxinstitute.org">Calyx Institute</a> for use in <a href="https://calyxos.org">CalyxOS</a></string>
-    <string name="about_source_code">Source code: https://github.com/seedvault-app/seedvault</string>
```

Comment:
And this as well.

Reply:
right, but then we can;t clutter the page. There's a link to github now already which should enable people to find the code from there.